internal/taskqueue/rabbitmq/rabbitmq.go

```diff
@@ -233,7 +230,10 @@ func (t *TaskQueueImpl) publish() {
 // }
 
 func (t *TaskQueueImpl) subscribe(ctx context.Context, subId, queue string, sessions chan chan session, messages chan *taskqueue.Task, tasks chan<- *taskqueue.Task) {
+	sessionCount := 0
+
 	for session := range sessions {
+		sessionCount++
```

Comment:
It's good to track session count for debugging, because high session values may indicate some issue with the rabbit connection.